src/sipnet/frontend.c

```diff
@@ -25,130 +26,144 @@
-  int numChangeableParams, i; // in pChange
-  int *indices; // indices of changeable params
-  char *paramName;  // name of one of the parameters that varies for a montecarlo run
-  double value; // one value from file
+  int numToSkip = 0;  // number of #'s to skip at start of each line of pChange
+  int statsOnly = 0;  // do we only output means & standard dev's for a
+                      // montecarlo run?
+  int numChangeableParams, i;  // in pChange
+  int *indices;  // indices of changeable params
+  char *paramName;  // name of one of the parameters that varies for a
+                    // montecarlo run
+  double value;  // one value from file
   char fileName[FILE_MAXNAME];
-  char outFile[FILE_MAXNAME+24];
-  char paramFile[FILE_MAXNAME+24], climFile[FILE_MAXNAME+24];
-  char mcParamFile[FILE_MAXNAME], mcOutFileBase[FILE_MAXNAME];  // used for runtype=montecarlo
+  char outFile[FILE_MAXNAME + 24];
+  char paramFile[FILE_MAXNAME + 24], climFile[FILE_MAXNAME + 24];
+  char mcParamFile[FILE_MAXNAME],
+      mcOutFileBase[FILE_MAXNAME];  // used for runtype=montecarlo
   int runNum;
 
-  // variables used for outputting means/standard deviations over a set of parameter sets:
+  // variables used for outputting means/standard deviations over a set of
+  // parameter sets:
   int j, k, type;
   double **model;
   double **means;
   double **standarddevs;
   int dataTypeIndices[MAX_DATA_TYPES];
 
 #if EVENT_HANDLER
-	// Extra filename if we are handing events
-	char eventFile[FILE_MAXNAME+24];
+  // Extra filename if we are handing events
+  char eventFile[FILE_MAXNAME + 24];
 #endif
 
   // get command-line arguments:
   while ((option = getopt(argc, argv, "hi:")) != -1) {
     // we have another optional argument
-    switch(option) {
-    case 'h':
-      usage(argv[0]);
-      exit(1);
-      break;
-    case 'i':
-      if (strlen(optarg) >= INPUT_MAXNAME)  {
-	printf("ERROR: input filename %s exceeds maximum length of %d\n", optarg, INPUT_MAXNAME);
-	printf("Either change the name or increase INPUT_MAXNAME in frontend.c\n");
-	exit(1);
-      }
-      strcpy(inputFile, optarg);
-      break;
-    default:
-      usage(argv[0]);
-      exit(1);
+    switch (option) {
+      case 'h':
+        usage(argv[0]);
+        exit(1);
+      case 'i':
+        if (strlen(optarg) >= INPUT_MAXNAME) {
+          printf("ERROR: input filename %s exceeds maximum length of %d\n",
+                 optarg, INPUT_MAXNAME);
+          printf("Either change the name or increase INPUT_MAXNAME in "
+                 "frontend.c\n");
+          exit(1);
+        }
+        strcpy(inputFile, optarg);
+        break;
+      default:
+        usage(argv[0]);
+        exit(1);
     }
   }
 
+  // clang-format off
   // setup namelist input:
   namelistInputs = newNamelistInputs();
-  addNamelistInputItem(namelistInputs, "RUNTYPE", STRING_TYPE, runtype, RUNTYPE_MAXNAME);
+  addNamelistInputItem(namelistInputs, "RUNTYPE", STRING_TYPE, runtype,RUNTYPE_MAXNAME);
```

Comment:
Another 🤨 -- it's _removing_ the space after just one of the commas on the line? What rule is driving this and is it actually  desirable?